_ml-commons-plugin/ml-dashboard.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: default
+title: Managing ML models in OpenSearch Dashboards
+nav_order: 120
+---
+
+Machine Learning in OpenSearch dashboards is an experimental feature released in OpenSearch 2.6. It can't be used in a production environment. For updates on the feature’s progress or to leave feedback on improving the feature, see the [OpenSearch Forum discussion](https://forum.opensearch.org/t/feedback-ml-commons-ml-model-health-dashboard-for-admins-experimental-release/12494).
+{: .warning }
+
+Administrators of machine-learning (ML) clusters can use OpenSearch Dashboards to manage and check the status of ML models running inside their cluster. This can aid in assisting ML developers with provisioning nodes to make sure their models run efficiently.
+
+As of OpenSearch 2.6, you can only upload models using the API. For more information about how to upload a model to your cluster, see [Upload model to OpenSearch]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-serving-framework#upload-model-to-opensearch)
+
+## Enabling Machine Learning in Dashboards
+
+In OpenSearch 2.6, Machine Learning is disabled by default. To enable it, you need to edit the configuration in `opensearch_dashboards.yml` and then restart your cluster.
+
+To enable the feature:
+
+1. In your OpenSearch cluster, navigate to your Dashboards home directory; for example, in Docker, `/usr/share/opensearch-dashboards`.
+2. Open your local copy of the Dashboards configuration file, `opensearch_dashboards.yml`. If you don't have a copy, get one from GitHub: [`opensearch_dashboards.yml`](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/config/opensearch_dashboards.yml).
+3. Add the setting `ml_commons_dashboards.enabled:`  to `opensearch_dashboards.yml`. Then. set it to  `ml_commons_dashboards.enabled: true` and save the configuration file.
+4. Restart the Dashboards container.
+5. Verify that the feature configuration settings were created and configured properly by launching OpenSearch Dashboards. The Machine Learning section should appear under **OpenSearch plugins**.
+
+## Accessing Machine Learning in Dashboards
+
+To access Machine Learning in OpenSearch Dashboards,select **OpenSearch plugins** > **Machine Learning**. 
+
+<img src="{{site.url}}{{site.baseurl}}/images/ml/ml-dashboard/ml-dashboard.png" alt="Machine Learning section in OpenSearch dashboards">
+
+Inside the Machine Learning section, you now have access to the **Deployed models** dashboard.
+
+## Deployed models dashboard
+
+The model health dashboards gives admins the ability to check the status of any models stored inside your OpenSearch cluster. 
```

Comment:
```suggestion
The model health dashboard, shown in the following image, gives admins the ability to check the status of any models stored inside an OpenSearch cluster. 
```

Reply:
The title and first sentence refer to this dashboard using different names. Should we use the same name in both instances?

Reply:
Switching to the "deployed models dashboard"